src/Providers/DataGrid/Wijmo/Columns/ImageColumn.ts

```diff
@@ -39,6 +39,7 @@ namespace Providers.DataGrid.Wijmo.Column {
             config.cellTemplate = Helper.CellTemplateFactory.MakeCellTemplate(
                 this.config.actionColumnElementType,
                 config.binding,
+                this.config.externalURL,
```

Comment:
We've discussed it before, but now I think this parameter should be optional in MakeCellTemplate function, and also placed after the altText parameter. This way we don't need to add it here, since it does not make sense in the ImageColumn.

Reply:
Done